Source/Scene/Cesium3DTileset.js

```diff
@@ -2340,7 +2341,13 @@ Cesium3DTileset.prototype.postPassesUpdate = function (frameState) {
   cancelOutOfViewRequests(this, frameState);
   raiseLoadProgressEvent(this, frameState);
   this._cache.unloadTiles(this, unloadTile);
-  this._styleEngine.resetDirty();
+
+  // If the style wasn't able to be applied this frame, keep it dirty so
+  // the engine can try to apply the style next frame.
```

Comment:
I would mention that a hidden tileset is one example of when the style wouldn't be applied.